server/src/resources/vertiport/mod.rs

```diff
@@ -80,6 +80,13 @@ impl Resource for GenericResource<Data> {
             ]),
         }
     }
+
+    fn get_table_indices() -> Vec<String> {
```

Comment:
Not sure why this was added. There's a default implementation of this function present in the Trait itself to return an empty list if no indices are needed (yet).

Reply:
Removed, was an artifact from trying to debug this error, which I still haven't solved:
```
arrow-svc-storage       | 2023-02-11 18:49:01 | INFO  | server/src/postgres/mod.rs:227 — Initializing global shared psql pool.
arrow-svc-storage       | Initializing database connection pool.
arrow-svc-storage       | 2023-02-11 18:49:01 | INFO  | server/src/postgres/mod.rs:154 — Setting up TLS connection with client cert and key.
arrow-svc-storage       | 2023-02-11 18:49:01 | INFO  | server/src/postgres/mod.rs:197 — Successfully created PostgresPool.
arrow-svc-storage       | 2023-02-11 18:49:01 | INFO  | server/src/postgres/mod.rs:924 — Creating database tables.
arrow-svc-storage       | 2023-02-11 18:49:01 | INFO  | server/src/postgres/mod.rs:255 — composing create table query for [vertiport]
arrow-svc-storage       | 2023-02-11 18:49:01 | INFO  | server/src/postgres/mod.rs:255 — composing create table query for [vertipad]
arrow-svc-storage       | 2023-02-11 18:49:01 | ERROR | server/src/postgres/mod.rs:311 — Failed to create indices for table [flight_plan]: db error: ERROR: there is no unique constraint matching given keys for referenced table vertiport
arrow-svc-storage       | 2023-02-11 18:49:01 | INFO  | server/src/postgres/mod.rs:255 — composing create table query for [vehicle]
arrow-svc-storage       | 2023-02-11 18:49:01 | ERROR | server/src/postgres/mod.rs:311 — Failed to create indices for table [flight_plan]: db error: ERROR: there is no unique constraint matching given keys for referenced table vertiport
arrow-svc-storage       | 2023-02-11 18:49:01 | INFO  | server/src/postgres/mod.rs:255 — composing create table query for [adsb]
arrow-svc-storage       | 2023-02-11 18:49:01 | INFO  | server/src/postgres/mod.rs:255 — composing create table query for [flight_plan]
arrow-svc-storage       | 2023-02-11 18:49:01 | ERROR | server/src/postgres/mod.rs:311 — Failed to create indices for table [flight_plan]: db error: ERROR: there is no unique constraint matching given keys for referenced table vertipad
arrow-svc-storage       | 2023-02-11 18:49:01 | INFO  | server/src/postgres/mod.rs:255 — composing create table query for [itinerary]
arrow-svc-storage       | 2023-02-11 18:49:01 | INFO  | server/src/postgres/mod.rs:227 — Initializing global shared psql pool.
```